src/Apps/Artist/Routes/CV/__test__/CV.test.tsx

```diff
@@ -0,0 +1,47 @@
+import { CV_Test_Query } from "__generated__/CV_Test_Query.graphql"
+import React from "react"
+import { graphql } from "react-relay"
+import {
+  MockBoot,
+  MockRelayRenderer,
+  renderUntil,
+} from "../../../../../DevTools"
+import {
+  CVFixture,
+  showsConnection,
+} from "../../../../__test__/Fixtures/Artist/CV"
+import { CVRouteFragmentContainer as CV } from "../../CV"
+
+jest.unmock("react-relay")
+
+describe("ArtistHeader", () => {
+  it("renders correct information about the artist", async () => {
+    const tree = await renderUntil(
+      wrapper => {
+        return wrapper.find(CV).length > 0
+      },
+      <MockBoot breakpoint="lg">
+        <MockRelayRenderer<CV_Test_Query>
+          Component={CV}
+          query={graphql`
+            query CV_Test_Query($artistID: String!) {
+              viewer {
+                ...CV_viewer
+              }
+            }
+          `}
+          mockResolvers={{
```

Comment:
@mzikherman - would you mind elaborating on this a bit? I might be running into a similar issue.

Reply:
Ah yeah, figured it out, was this. I like how it stubs things out with Hello World so nothing blows up but until you start digging into the data it can be a bit deceptive. Thought the top level mock resolver (`Viewer`) would cover all of the resolved sub deps. Might be somewhat confusing for others. 

Does it make sense to go over to `relay-mock-network` to add some kind of warning for when data was being stubbed? cc @alloy

Reply:
Yeah that’s what I intended by https://artsy.slack.com/archives/C2TQ4PT8R/p1540417353000100. I.e. we should have createMockNetworkLayer override these for all tests

Reply:
@mzikherman Was there any reason in the end to still not use renderRelayTree?

Reply:
(re `renderRelayTree`, I ended up updated this in my other PR)